go.mod

```diff
@@ -5,14 +5,15 @@ go 1.22
 toolchain go1.22.0
 
 require (
-	github.com/eapache/go-resiliency v1.6.0
```

Comment:
Is this change related?

Reply:
Deps update is unrelated and can be sent separately as a standalone PR.

Let me know if there is such thing is needed.